docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: .
       args:
-        PMM_SERVER_IMAGE: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-container}
+        PMM_SERVER_IMAGE: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
       dockerfile: devcontainer.Dockerfile
```

Comment:
Why don't just use `image` field instead of `build`?

Reply:
Idk. Here's [changes related](https://github.com/percona/pmm/commit/0dfd5220b1b1a1c5262e20dacffddb4f1160faf5)

Reply:
let's revert changes related to the docker-compose file and make your changes in the commented part.